modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -25,24 +25,26 @@ struct ONNXInitPath {
     }
 };
 static ONNXInitPath g_init_path;
-
-cv::Mat initMatrixRandU(const int type, const cv::Size& sz_in) {
-    const cv::Mat in_mat1 = cv::Mat(sz_in, type);
-
-    if (CV_MAT_DEPTH(type) < CV_32F) {
-        cv::randu(in_mat1, cv::Scalar::all(0), cv::Scalar::all(255));
-    } else {
-        const int fscale = 256;  // avoid bits near ULP, generate stable test input
-        cv::Mat in_mat32s(in_mat1.size(), CV_MAKE_TYPE(CV_32S, CV_MAT_CN(type)));
-        cv::randu(in_mat32s, cv::Scalar::all(0), cv::Scalar::all(255 * fscale));
-        in_mat32s.convertTo(in_mat1, type, 1.0f / fscale, 0);
-    }
-    return in_mat1;
-}
 } // anonymous namespace
 namespace opencv_test
 {
 namespace {
+void initTestDataPath()
+{
+#ifndef WINRT
+    static bool initialized = false;
+    if (!initialized)
+    {
+        // Since G-API has no own test data (yet), it is taken from the common space
+        const char* testDataPath = getenv("OPENCV_TEST_DATA_PATH");
+        GAPI_Assert(testDataPath != nullptr);
+
+        cvtest::addDataSearchPath(testDataPath);
+        initialized = true;
+    }
+#endif // WINRT
+}
+
 // FIXME: taken from the DNN module
 void normAssert(const cv::InputArray& ref, const cv::InputArray& test,
```

Comment:
> const cv::InputArray&

`InputArray` is already a const reference.

Reply:
Fixed.